### PR TITLE
Create spam_image_hidden_element.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description
Adding detection for emails with observed body html template that uses a single image as body content and hides the rest of the html.

# Associated samples
- [Sample 1](https://platform.sublimesecurity.com/messages/c4f3fe0398d704f0d88db9988ca5d85cc2452ae74581b2c3492a2b9bb72da121)
- [Sample 2](https://platform.sublimesecurity.com/messages/e9feb03520daad74e2fe1dcda262cbb0b54c68ab012a41dd1cc6edf27e1a28b3)
- [Sample 3](https://platform.sublimesecurity.com/messages/00d31f1821cd55ea499d35b7df6c0f89f3b64acb9909b1ffe4dea02fe43f49c2)

## Associated hunts
- [Hunt 1](https://platform.sublimesecurity.com/hunts/a34d973e-74f6-443f-9dae-ed983063cd99)

### CODE REVIEW

**Comment by zoomequipd** [comment]
baking

**Comment by zoomequipd** [comment]
still baking

**Comment by zoomequipd** [comment]
looking ok, not a lot of volume, but all TPs

File: detection-rules/spam_image_hidden_element.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                      '<a href="https?:\/\/[^\x22]+\x22><img\s*src=\x22https:\/\/[^\x22]+\x22\><\/a>(?:<\/?[^>]*>){0,3}<(?:span|div)\s*style=\x22\s*display\s*\x3a\s*none\x3b[^\x22]*\x22>(?:_|&nbsp\x3b\x200c?){3,}'
+  )
+
+attack_types:
+  - "Spam"
+tactics_and_techniques:
+  - "Evasion"
+  - "Image as content"
+detection_methods:
+  - "Content analysis"
```
Comment (morriscode) [inline detection-rules/spam_image_hidden_element.yml:27]:
```suggestion
  - "Content analysis"
  - "HTML analysis"
  - "Sender analysis"
```

**Review by morriscode — CHANGES_REQUESTED** [review]
Left a suggestion, other than that looks great!

**Review by morriscode — APPROVED** [review]

**Review by morriscode — APPROVED** [review]

**Comment by zoomequipd** [comment]
looking good. pushing.

**Pull request merged**